app/Http/Controllers/SearchController.php

```diff
@@ -6,12 +6,28 @@
 use App\Watchlist;
 use App\Review;
 use App\Comment;
+use App\Services\Client;
 
 class SearchController extends Controller
 {
+    private $_client;
+
+    public function __construct(Client $_client)
+    {
+        $this->client = $_client;
+    }
     public function view()
     {
-        $client = new \GuzzleHttp\Client();
+        $client = $this->client;
+        // dd($client, app(), 'yolo');
```

Comment:
Some leftover comments, might as well remove now. Just in case

Reply:
:+1: